client/lib/add-query-args/README.md

```diff
@@ -0,0 +1,16 @@
+addQueryArgs
```

Comment:
Perhaps in https://github.com/Automattic/wp-calypso/blob/master/client/lib/route/path.js